STANDARDS.md

```diff
@@ -31,6 +31,11 @@
   * Follow [Dockerflow specification](https://github.com/mozilla-services/Dockerflow)
   * Use third party development tools for code coverage, documentation, etc.
 
+* **DOs and DONTs**
+
+  * **DO** push your `git` topic branches to **your fork** (not `mozilla`) and make
```

Comment:
I don't really care for this model. If the rest of the team votes to fork+pr back I have no objects.

Reply:
Why don't you care for it, @mostlygeek? It has served well on all the other projects I've worked on. Have you had problems with it, or seen benefits on other other projects doing something else?

Reply:
I believe the checkbox @magopian mentions is on by default. I've also only used that feature a couple times since it launched.